rust/xaynet-analytics/src/database/analytics_event/data_model.rs

```diff
@@ -65,3 +66,22 @@ impl IsarAdapter for AnalyticsEvent {
         object_builder.write_string(Some(&self.timestamp.to_rfc3339()));
     }
 }
+
+impl Display for AnalyticsEvent {
```

Comment:
Why we need this?

Reply:
[because of this](https://github.com/xaynetwork/xaynet/pull/709/files#r568586409)